contracts/test/CW20toERC20PointerTest.js

```diff
@@ -152,6 +152,27 @@ describe("CW20 to ERC20 Pointer", function () {
                     const balanceAfter = respAfter.data.balance;
                     expect(balanceAfter).to.equal((parseInt(balanceBefore) - 100).toString());
                 });
+
+                it("should be callable via wasmd precompile", async function() {
+                    const respBefore = await queryWasm(pointer, "balance", {address: accounts[1].seiAddress});
+                    const balanceBefore = respBefore.data.balance;
+
+                    const receipt = await callWasmViaPrecompile(ethers.provider, pointer, { transfer: { recipient: accounts[1].seiAddress, amount: "100" } });
+                    const respAfter = await queryWasm(pointer, "balance", {address: accounts[1].seiAddress});
+                    const balanceAfter = respAfter.data.balance;
+
+                    expect(balanceAfter).to.equal((parseInt(balanceBefore) + 100).toString());
+
+                    // make sure log is emitted
+                    console.log(receipt);
+                    const filter = {
+                        fromBlock: receipt["blockNumber"],
+                        toBlock: 'latest',
+                        topics: [ethers.id("Transfer(address,address,uint256)")]
+                    };
+                    const logs = await ethers.provider.getLogs(filter);
+                    expect(logs.length).to.equal(1)
```

Comment:
is it possible to string this test together to see a the set of logs unified from both evm invocations?(>1)

Reply:
it's a little difficult since the wasmd precompile doesn't emit event, and we don't allow pointer pointing to another pointer